spec/features/cases/foi/case_approvals/disclosure_specialist_approval_spec.rb

```diff
@@ -72,19 +73,19 @@ def undo_de_escalate_case_as_disclosure_specialist(kase, case_list_item)
     expect(kase.reload.approving_teams).to include team_dacu_disclosure
   end
 
-  def create_flagged_case_and_assign_to_team(days_back)
-    login_as manager
-    cases_page.load
-    cases_page.new_case_button.click
+  # def create_flagged_case_and_assign_to_team(days_back)
+  #   login_as manager
+  #   cases_page.load
+  #   cases_page.new_case_button.click
```

Comment:
let's remove if elsewhere now.

Reply:
✓

Reply:
I've switched these features to use the feature steps that are more generally available.